.github/workflows/test.yaml

```diff
@@ -82,18 +80,16 @@ jobs:
           ~/.vvm
         key: compiler-cache
 
-    - name: Cache compiler installations
+    - name: Cache Brownie Build Folders
       uses: actions/cache@v2
       with:
         path: |
-          ~/.solcx
-          ~/.vvm
```

Comment:
why did you remove these? they make sense

Reply:
it's in 2 places